### PR TITLE
Updates pre-commit order

### LINKED ISSUE #157: Change the order of pre-commit check
Today prospector is the first, but it's slow on larger projects.
Better to have the following order: isort, eslint, missing-migrations, prospector.

### PR DESCRIPTION
**Pull request by rsarai**
Resolves #157

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires dependencies updates.
- [ ] I have updated the dependencies accordingly.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -31,16 +31,16 @@
...
-    pass_filenames: true
   - id: missing-migrations
     name: missing-migrations-local
     entry: python manage.py has_missing_migrations
     language: system
     always_run: true
     pass_filenames: false
+  - id: prospector
+    name: prospector-local
+    entry: prospector
```
Comment (chocoelho) [inline .pre-commit-config.yaml]: Add this to prospector entry: `prospector --messages-only --zero-exit`. It'll return only linter messages and 0 if everything went okay. It provides a clearer output.

**Comment by chocoelho** [comment]
zero-exit is only if everything went okay, other than that it'll return 
the error

File: .pre-commit-config.yaml (hunk 1 above)
Comment (chocoelho) [reply .pre-commit-config.yaml]: https://prospector.landscape.io/en/master/usage.html#Named%20Arguments

**Comment by aericson** [comment]
```
> prospector .  --messages-only
Messages
========

users.tests.__init__ (users/tests/__init__.py):
    L1:- None: pylint - cyclic-import
    Cyclic import (redis_subscriber.consumer_actions -> redis_subscriber.consumers)

(sensus_dashboard-rPd116tv) ➜  sensus_dashboard git:(master) ✗ echo $?
1
> prospector .  --messages-only --zero-exit
Messages
========

common.utils.tests (common/utils/tests.py):
    L1:- None: pylint - cyclic-import
    Cyclic import (redis_subscriber.consumer_actions -> redis_subscriber.consumers)


> echo $?
0
```

Wouldn't that cause the pre-commit to pass?

**Comment by chocoelho** [comment]
@aericson it's a problem with how prospector is documented: https://github.com/landscapeio/prospector/issues/177#issuecomment-232839815

Leave `--zero-exit` out.

**Comment by chocoelho** [comment]
@rsarai any updates on this?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (fjsj) [reply .pre-commit-config.yaml]: Can you fix this and merge @rsarai ?

**Pull request merged**